src/main/java/com/chopshop166/chopshoplib/outputs/SmartSpeedController.java

```diff
@@ -97,18 +95,15 @@ public void pidWrite(double output) {
     /**
      * Create an instance of this interface that dispatches to the wrapped objects.
      * <p>
-     * This can be used to wrap a group of objects that implements the two base
-     * interfaces separately, but not this interface. Motors can be passed and a
-     * speed controller group is created automatically.
+     * This can be used to wrap a group of speed controllers. Motors can be passed
+     * and a speed controller group is created automatically.
      * 
      * @param motor  Motor that will be wrapped, will be {@link Sendable} and put
      *               into a {@link SpeedControllerGroup}.
      * @param motors Continuation of motor.
      * @return A wrapped Speed Controller Group.
      */
-    static SendableSpeedController group(SpeedController motor, SpeedController... motors) {
-        SpeedControllerGroup motorGroup = new SpeedControllerGroup(motor, motors);
-
-        return SendableSpeedController.wrap(motorGroup);
+    static SmartSpeedController group(SpeedController motor, SpeedController... motors) {
```

Comment:
It might be super cool to have this set up followers for the smart controllers. Totally a stretch goal though